app/models/account.rb

```diff
@@ -289,6 +289,16 @@ def protected?
     protected
   end
 
+  def sso? = sso_organization_id?
```

Comment:
Should we also check `ent?`, or should we let this be open in case we sell an SSO add-on for Std tiers too?